src/meta/src/hummock/hummock_manager.rs

```diff
@@ -1110,6 +1114,14 @@ where
 
         // Update metrics
         trigger_commit_stat(&self.metrics, &versioning.current_version);
+        for compaction_group_id in &modified_compaction_groups {
```

Comment:
Since we only check on new epoch, will we also run into the issue where if the sources have stopped, then we will also see stale metrics?

I guess the scenario we are worried about is 
1. Compactors are all dead
2. Sources not producing = no new epoch

Then we will not see any updated SST metrics? Is this even a problem?

Reply:
If there is no data written into storage and no compaction running, our LSM should not change so there is no need to update the LSM metrics in such cases.